src/controllers/stocks/index.ts

```diff
@@ -70,6 +74,10 @@ export const stocksRouter = async function (app: FastifyInstance) {
 		},
 		async (req) => {
 			const { server_id, user_id, symbol, amount } = req.body;
+			if (!symbol || !amount)
```

Comment:
this will throw if `amount` is 0, even if a minimum value is set in the validator, this is dangerous practice

Reply:
In this case we want it to throw if `amount` is 0.

Reply:
And it appears to be hitting the min value validation first anyway:
![image](https://user-images.githubusercontent.com/1410481/169719968-0f40c1de-24fd-43d9-80c7-7bdcdb3b078f.png)

Reply:
if its set in the validator already then that check would never even hit

Reply:
[Fix](https://github.com/BillyP-Bot/billybot-backend/pull/7/commits/ebf7155d41820db305e4d991370651ecec22c7f0)

![image](https://user-images.githubusercontent.com/1410481/169720554-fb9edb55-a320-4b22-b371-3f231f711daa.png)